.gitignore

```
@@ -1,3 +1,4 @@
 .idea
 .vscode
 *.log
+build/
```

probably a good idea to make this
```suggestion
/build/
```
so it only matches in the root